doc/hacking.rst

```diff
@@ -527,3 +527,36 @@ Good:
 .. code:: ocaml
 
    let (foo : _ Command.Args.t) = S []
+
+Benchmarking
+============
+
+Dune Bench
+----------
+
+You can benchmark Dune's performance by running `make bench`. This will run a
+subset of the duniverse.
```

Comment:
```suggestion
subset of the Duniverse.
```

does ‘make bench’ need two ‘ on either side to make it monospace in .rst files?

Reply:
That was supposed to be a backtick my bad.